sygnal/sygnal.py

```diff
@@ -76,6 +76,17 @@ def __init__(self, config, custom_reactor, tracer=opentracing.tracer):
         observer = twisted_log.PythonLoggingObserver()
         observer.start()
 
+        # Old format db config
+        if config.get("db", None) is not None:
+            logger.warning(
+                "Config includes the legacy 'db' option, please migrate"
+                + "to 'database' instead"
```

Comment:
"migrateto" is two words. `+` operator is redundant.

```suggestion
                " to 'database' instead"
```